test/compilable/deprecations.d

```diff
@@ -0,0 +1,49 @@
+/*
+REQUIRED_ARGS: -dw
+TEST_OUTPUT:
+--
+compilable\deprecations.d(37): Deprecation: struct deprecations.S is deprecated
+compilable\deprecations.d(37): Deprecation: struct deprecations.S is deprecated
+compilable\deprecations.d(39): Deprecation: struct deprecations.S is deprecated
+compilable\deprecations.d(39): Deprecation: struct deprecations.S is deprecated
```

Comment:
Why the double message ?

Reply:
`dsymbolSemantic` is run multiple times for the variable/parameters for some reason but this behaviour is already present with current master/release